pokemon_battle_arena/src/db/moves.rs

```diff
@@ -88,7 +86,7 @@ impl Technique {
                             (self.get_name() == String::from("synthesis")) ||
                             (self.get_name() == String::from("morning-sun")) {
```

Comment:
You can compare a `String` with a `&str`. There is no need to say `String::from("...")`. `self.get_name() == "synthesis"` should work as well. 

Also: this looks like a stringly-typed API which is absolutely not idiomatic in Rust. Assume you notice a typo in "morning-sun". Now you would change the typo in its definition somewhere, but you forget to fix the typo in this line. Now you have a bug which is really hard to detect. What if you had used enums instead of strings (something like `self.get_move() == Move::MorningSun`)? Then the compiler would have shown an error and thus avoided the bug. 

Again: use the compiler to your advantage, don't fight it.